### PR TITLE
Corda journal

### PR DESCRIPTION
**Pull request by zhengmar3**
Added Corda transaction table.

1. Integration with **_react-query_** with fake/dummy data.
2. Use of MUI **_datagrid_** component
3. Table pagination, hide/show columns, filtering, table density changeability is integrated.
4. Function to export/download the table data as .csv file or print also integrated.
5. Latest commit added quick search function to the table 

basic display:
image

Show/hid columns:
image

filtering:
image

export/download:
image

quick search:
image

### CODE REVIEW

File: mui-template-app/src/components/AppRoutes/AppRoutes.tsx [hunk 1]
```diff
@@ -28,6 +29,7 @@ function AppRoutes() {
         <Route path="react-query" element={<ReactQuery />} />
         <Route path="file-upload" element={<FileUploadSamples />} />
         <Route path="data-table" element={<DataTableExample />} />
+        <Route path="corda-transaction" element={<CordaTransaction />} />
```
Comment (yarivus) [inline mui-template-app/src/components/AppRoutes/AppRoutes.tsx:32]: Maybe we should rename to CordaTransactionTable.
In the future we might have different representation (e.g. CordaTransactionAudit/Log).

**Review by yarivus — COMMENTED** [review]
Nice @zhengmar3 - see my question/comment

File: mui-template-app/src/components/AppRoutes/AppRoutes.tsx (hunk 1 above)
Comment (zhengmar3) [reply mui-template-app/src/components/AppRoutes/AppRoutes.tsx]: Updated.

**Review by yarivus — APPROVED** [review]
LGTM

**Pull request merged**